source/agora/test/Base.d

```diff
@@ -245,7 +245,7 @@ public class TestNetworkManager : NetworkManager
     import geod24.LocalRest;
 
     /// Workaround compiler bug that triggers in `std.concurrency`
-    protected __gshared std.concurrency.Tid[string] tbn;
+    public __gshared std.concurrency.Tid[string] tbn;
```

Comment:
Same thing about `protected`

Reply:
Well `BadAPIManager` is not a subclass of `TestNetworkManager`, but of `TestAPIManager`, so it needs to be public. 

This is after I split the two classes into `TetAPIManager` and `TestNetworkManager`. I didn't feel it made sense to have it one class just in order to keep one `__gshared` field `protected.`

Reply:
Fair enough